src/screens/TopicMemberScreen/index.js

```diff
@@ -48,35 +40,58 @@ const TopicMemberScreen = () => {
   const [profile] = React.useContext(Context).profile;
   const [isInitialLoading, setIsInitialLoading] = React.useState(true);
   const [headerHide, setHeaderHide] = React.useState(false);
-
+  const [searchHeight, setSearchHeight] = React.useState(0);
   const [searchText, setSearchText] = React.useState('');
   const [initalMember, setInitialMember] = React.useState([]);
   const [topicDataFollowedUsers, setTopicDataFollowedUsers] = React.useState([]);
   const [topicDataUnfollowedUsers, setTopicDataUnfollowedUsers] = React.useState([]);
   const [isFocus, setIsFocus] = React.useState(true);
   const [isFirstTimeOpen, setIsFirstTimeOpen] = React.useState(true);
   const cancelTokenRef = React.useRef(axios.CancelToken.source());
+  const scrollY = React.useRef(new Animated.Value(0)).current;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of new state variables like `searchHeight`, `isFocus`, `isFirstTimeOpen` indicates that new features or changes have been introduced. Ensure these variables are used appropriately throughout the component.



<!-- This is an auto-generated comment by OSS CodeRabbit -->